### ISSUE TITLE
Drop the blog and just use a medium publication

### ISSUE THREAD

**Issue by paulfalgout** [marionettejs/blog#39]
- Medium is way better for authoring and drafts.. 
- It's where readers are
- we can get a custom domain so just point blog.marionettejs.com to medium
- we can request other marionette or even backbone posts be published by our publication with a click.
- anyone can submit just like on here except they can still publish it if we turn it down (for some reason)
- no maintenance

**Comment by rafde** [comment]
👍
You can also draft and have others review before publishing.
Easy and seamless to revise published post.

**Comment by blikblum** [comment]
The only drawback of medium IMO is the lack of source highlight. Is necessary to embed gists. Anyway the benefits outweigh this issue

**Comment by denar90** [comment]
Nice. Should we also copy-past old blog posts into medium then?

**Comment by scott-w** [comment]
I agree with @blikblum in that we have to embed gists instead of just using source highlighting.

Does Medium have a good way to get feedback on our posts? I can definitely see the value in knowing what things we published were popular and people, potentially, want to see more of in future.

**Comment by paulfalgout** [comment]
For sure.. comments / recommends and then stats on reads and read ratio.  Certainly a lot more than we have now.
<img width="1034" alt="screen shot 2017-01-11 at 10 19 39 pm" src="https://cloud.githubusercontent.com/assets/2028470/21850041/11fe05ec-d84c-11e6-8994-c985ae90cc31.png">

**Comment by denar90** [comment]
Guys what do we wanna make first? Setup `Publications page` ? @scott-w do you wanna lead this? I don't have needed credentials for domain and other stuff...

And still what are your thoughts about old posts? Will we move them to medium?
